test/suites/base__fedprops/base__fedprops.conf.js

```diff
@@ -1,3 +1,3 @@
 'use strict';
 
-exports.config = { suite: [ './specs/fedprops/*.js' ] };
+exports.config = { suite: [ '../specs/fedprops/*.js' ] };
```

Comment:
I think this change is not necessary if the way these strings are interpreted is changed in 'wdio.conf.js`. I'd prefer to change it over there, because this path now is counter intuitive.

Reply:
Correct!